yarn.lock

```diff
@@ -241,6 +261,27 @@
     semver "^6.3.0"
     source-map "^0.5.0"
 
+"@babel/core@^7.18.2":
```

Comment:
`@linaria/babel-preset@^3.0.0-beta.22` depends on `@babel/core@^7.18.2` -> adding yet another `@babel/core` version :/

Reply:
you can merge the above `@babel/core` entry with the new one, I'm pretty sure

```
"@babel/core@^7.1.0", "@babel/core@^7.10.4", "@babel/core@^7.12.10", "@babel/core@^7.12.13", "@babel/core@^7.12.3", "@babel/core@^7.7.2", "@babel/core@^7.7.5"
```

all resolves to the new version

Reply:
`npx yarn-deduplicate --scopes @babel` should do the thing, https://www.npmjs.com/package/yarn-deduplicate.

Reply:
yeah give that one a try, I see that @babel/generator can also be deduped